src/walletinitinterface.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef WALLETINITINTERFACE_H
+#define WALLETINITINTERFACE_H
+
+class CScheduler;
+class CRPCTable;
+
+class WalletInitInterface {
```

Comment:
In commit "[wallet] Create wallet init interface."

Just FYI, this PR is very similar to 6626e2fe114ce2aa861be1eeda84f0c9faa60527 from #10973 and the `WalletInitInterface` added here is basically identical to the `Chain::Client` interface added there:

https://github.com/bitcoin/bitcoin/blob/6626e2fe114ce2aa861be1eeda84f0c9faa60527/src/interface/chain.h#L18-L40

Probably would replace this interface with that more generic one if this is merged first.

Reply:
Thanks for the heads up!